docs/src/Module.doc.js

```diff
@@ -352,16 +351,18 @@ function ModuleExample5() {
       'second-1': 1,
   }
   return (
-    <Box maxWidth={800} padding={2} column={12}>
-      <Flex direction='column' gap={3}>
-        <Box padding={1} borderStyle='sm'>
-          <Text>Step 1</Text>
+    <Box column={12} maxWidth={800} padding={2}>
+      <Flex direction="column" gap={4}>
+        <Flex direction="column" gap={2}>
```

Comment:
I reworked this a bit to both simplify the existing markup, and also prettify the example UI a bit. The border was weird, spacing far too tight, etc